### ISSUE TITLE
problem with displaying in-transit items?

### ISSUE THREAD

**Issue by theLinkResolver** [ualbertalib/discovery#813]
I can't open the record for Neo the NEOS Bear. I noted that his current location is INTRANSIT.

https://search.library.ualberta.ca/results?utf8=%3F&q=neo+bear

**Comment by theLinkResolver** [comment]
Okay, just noticed this is shadowed at the item level. This is different from when a shadowed location is used (e.g. IN_PROCESS); there is a checkbox available in the item record to shadow an item. This is rare but it happens. @redlibrarian Not a showstopper by any stretch but there are a few of these out there for sure. We don't want to see them at all. I have renamed the issue accordingly.

**Comment by redlibrarian** [comment]
@theLinkResolver Hmm. The Symphony extract logic is supposed to exclude any bib records for which all items have been shadowed (ie. even if the bib record isn't shadowed, if it doesn't have any visible items, it should be excluded). I guess the logic of the extract isn't working perfectly - I'll have to talk to Jim about it...

**Comment by theLinkResolver** [comment]
@redlibrarian I should also mention there is a way to shadow at the title level, too (another checkbox - i.e. Yes/No setting). Might as well make sure all bases are covered in your convo with Jim.

**Comment by redlibrarian** [comment]
@theLinkResolver Yes, that's what I mean by "bib level". I'll check with Jim in the new year.

**Comment by redlibrarian** [comment]
Is that Neos Bear record still in the system? If not, is there another record I can test with? / @theLinkResolver

**Comment by theLinkResolver** [comment]
The bear is still here: 6734650 - shadowed at the item level, using a checkbox

This record has a single item in a shadowed location (DISCARD): 5112298

This record is shadowed at the bib level, using a checkbox: 2892680

**Comment by redlibrarian** [comment]
From what I can see in search-test:
6734650: is not in the index
5112298: is not in the index
2892680: is in the index, but pulls back no holdings.

I think after the launch, I'll have to work with Jim to see if we can exclude the "checkbox" shadowed records.

**Comment by redlibrarian** [comment]
Done when:

- [ ] Shadowed records excluded from index
